Geometry/Area/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+
+public class Rectangle {
+	private double height;
+	private double width;
+	private double area;
+	
+	public Rectangle(double height, double width) {
+		this.height = height;
+		this.width = width;
+	}
+	
+	public double getHeight() {
+		return height;
+	}
+
+	public double getWidth() {
+		return width;
+	}
+
+	public double getArea() {
+		return area;
+	}
+
+	public void setArea() {
+		this.area = AreaFormulas.rectangle(height, width);
```

Comment:
This method isn't needed. Just return the calculated value in getArea()

Reply:
Deleted the whole class. I didn't go that route in the end.